src/main/java/de/tum/in/www1/artemis/service/connectors/athena/AthenaFeedbackSendingService.java

```diff
@@ -0,0 +1,122 @@
+     * @param athenaRestTemplate  The rest template to use for sending requests to Athena
+     */
+    public AthenaFeedbackSendingService(TextBlockRepository textBlockRepository, @Qualifier("athenaRestTemplate") RestTemplate athenaRestTemplate) {
+        connector = new AthenaConnector<>(log, athenaRestTemplate, ResponseDTO.class);
+        this.textBlockRepository = textBlockRepository;
+    }
+
+    private static class RequestDTO {
+
+        public TextExerciseDTO exercise;
+
+        public TextSubmissionDTO submission;
+
+        public List<TextFeedbackDTO> feedbacks;
+
+        /**
+         * Connect feedback and text block to find the correct start and end indexes for transfer when constructing the DTO:
+         */
+        RequestDTO(@NotNull TextExercise exercise, @NotNull TextSubmission submission, @NotNull List<Feedback> feedbacks, TextBlockRepository textBlockRepository) {
+            this.exercise = TextExerciseDTO.of(exercise);
+            this.submission = TextSubmissionDTO.of(exercise.getId(), submission);
+            this.feedbacks = feedbacks.stream().map(feedback -> {
+                // Give the DTO the text block the feedback is referring to.
+                // => It will figure out start and end index of the feedback in the text
+                var feedbackTextBlock = textBlockRepository.findById(feedback.getReference()).orElse(null);
+                return TextFeedbackDTO.of(exercise.getId(), submission.getId(), feedback, feedbackTextBlock);
+            }).toList();
+        }
+    }
+
+    private record ResponseDTO(String data) {
+    }
+
+    /**
+     * Calls the remote Athena service to submit feedback given by a tutor
+     *
+     * @param exercise   the exercise the feedback is given for
+     * @param submission the submission the feedback is given for
+     * @param feedbacks  the feedback given by the tutor
+     */
+    public void sendFeedback(TextExercise exercise, TextSubmission submission, List<Feedback> feedbacks) {
+        sendFeedback(exercise, submission, feedbacks, 1);
+    }
+
+    /**
+     * Calls the remote Athena service to submit feedback given by a tutor
+     *
+     * @param exercise   the exercise the feedback is given for
+     * @param submission the submission the feedback is given for
+     * @param feedbacks  the feedback given by the tutor
+     * @param maxRetries number of retries before the request will be canceled
+     */
+    public void sendFeedback(TextExercise exercise, TextSubmission submission, List<Feedback> feedbacks, int maxRetries) {
+        if (!exercise.isFeedbackSuggestionsEnabled()) {
+            throw new IllegalArgumentException("The Exercise does not have feedback suggestions enabled.");
+        }
+
+        log.debug("Start Athena Feedback Sending Service for Text Exercise '{}' (#{}).", exercise.getTitle(), exercise.getId());
+
+        if (feedbacks.isEmpty()) {
+            log.info("No feedback given for submission #{}.", submission.getId());
```

Comment:
I don't think this is an actual information.
```suggestion
            log.debug("No feedback given for submission #{}.", submission.getId());
```